tests/php/Control/SessionHandler/DatabaseSessionHandlerTest.php

```diff
@@ -0,0 +1,275 @@
+<?php
+
+namespace SilverStripe\Control\Tests\SessionHandler;
+
+use PHPUnit\Framework\Attributes\DataProvider;
+use ReflectionMethod;
+use SilverStripe\Control\Session;
+use SilverStripe\Control\SessionHandler\DatabaseSessionHandler;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\ORM\DB;
+use SilverStripe\ORM\FieldType\DBDatetime;
+
+class DatabaseSessionHandlerTest extends SapphireTest
+{
+    protected $usesDatabase = true;
+
+    protected static $fixture_file = 'DatabaseSessionHandlerTest.yml';
+
+    public function onBeforeLoadFixtures(): void
+    {
+        // Add the sessions table
+        $handler = new DatabaseSessionHandler();
+        DB::get_schema()->schemaUpdate(fn () => $handler->requireTable());
```

Comment:
```suggestion
        DB::get_schema()->schemaUpdate(fn() => $handler->requireTable());
```

Reply:
Done - if the linter complains I'll undo it